texada-src/src/main/texadamain.cpp

```diff
@@ -336,9 +338,9 @@ int main(int ac, char* av[]) {
                         outFile << ", ";
                     print_json(outFile, *it, opts_map.count("print-stats"));
                 } else {
-                    const spot::ltl::formula * valid_form = texada::instantiate(
+                    const ltl::formula * valid_form = texada::instantiate(
```

Comment:
If we're using namespace texada, why do we need texada::instantiate rather than just instantiate?

Reply:
This is true, we can use just instantiate.